core/builder/src/main/java/io/quarkus/builder/JsonTransform.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.builder;
+
+import java.util.function.Predicate;
+
+@FunctionalInterface
+public interface JsonTransform {
+    void accept(Json.JsonBuilder<?> builder, JsonReader.JsonValue element);
```

Comment:
Again naming in line with Class-File API, but I'm curious, why the name `keep`?

Reply:
Because of `dropping`. One of the keeps and the other one drops, but you essentially override `keep` in `dropping` so it doesn't make sense. +1 on following the Class-File API. Sorry for the noise.

Reply:
Right, `dropping` is just one type of transformation you can implement with this. IOW, `dropping` implements `accept` to drop certain JSON. For this particular use case, it's the only high level transformation I required, but you could other ones that transform different parts of the JSON.